R/est.incidence.by.R

```diff
@@ -101,6 +101,27 @@ est.incidence.by <- function(
     return(to_return)
   }
 
+  if (!any(is.element(strata,pop_data %>% names()))) {
```

Comment:
please always format according to the style guide (add a space after the comma in `strata,pop_data`, etc)

Reply:
The change has been made.